Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -694,6 +702,23 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
         this._showPrintStylesChanged();
     }
 
+    _handleOverrideUserPreferencesButtonClicked()
+    {
+        if (this._userPreferencesOverridesPopover) {
+            this._userPreferencesOverridesPopover.dismiss();
+            this._userPreferencesOverridesPopover = null;
```

Comment:
Won't this be done by the delegate function `didDismissPopover(popover)`?

Reply:
Yes, `null`-ing the popover instance happens in `didDismissPopover()` but that occurs after a CSS `transitionend` event that takes a few hundred milliseconds to trigger. See [Popover.js?L189-200](https://sourcegraph.com/github.com/WebKit/WebKit@f7c30648762020ca176afcd3932b878a07b44994/-/blob/Source/WebInspectorUI/UserInterface/Views/Popover.js?L189-200)

`this._handleOverrideUserPreferencesButtonClicked()` reacts to clicks on the trigger element. There's a chance the user clicks repeatedly and very quickly. `null`-ing immediately here saves the trouble with potential zombie instances living long enough to cause issues.